subjects/binaryaddition/README.md

```diff
@@ -0,0 +1,44 @@
+## binary-addition
+
+### Instructions
+
+Write a function named `BinaryAddition(int,int)` that takes two integers and returns the sum of the two in binary in an array of `int`.
+- If the sum is negative return `nil` value.
+- convert the argument to binary then add the two binary numbers together
+
+### Expected function 
+```go
+func BinaryAddition(a int, b int) []int {
+    // your code here
+}
+```
+
+### Usage
+
+```go
+package main
+
+import "fmt"
+
+
+func main(){
+	fmt.Println(BinaryAddition(1, 1))
+	fmt.Println(BinaryAddition(1, 2))
+	fmt.Println(BinaryAddition(1, 3))
+	fmt.Println(BinaryAddition(2, 1))
+	fmt.Println(BinaryAddition(2, 2))
+	fmt.Println(BinaryAddition(2, 3))
+}
+```
+
+and the output should be:
+
+```console
+$ go run . 
+[1 0]
```

Comment:
Well done, but you can remove those square bracket  since there are not included in the Code Output

Reply:
actually, when we use `fmt.Println` for print array, the output print like that.